tests/libtest/Makefile.inc

```diff
@@ -34,7 +34,7 @@ noinst_PROGRAMS = chkhostname libauthretry libntlmconnect                \
  lib1558 \
  lib1560 \
  lib1591 lib1592 \
- lib1900 lib1905 lib1906 \
+ lib1900 lib1905 lib1906 lib1942 \
```

Comment:
You add a lib1942 tool that gets built, but no test that uses it?

Reply:
Yes, sorry. I wrote client-site code, but stuck on perl api, because i need https server. 
Is it nesessary to add test to this PR? I think that it is easier to just to remove lib1942 code.

Reply:
lib1942 file was removed